yearbook2023.md

```diff
@@ -3,6 +3,7 @@
 Sign our yearbook! Add your totally awesome team name and your favorite VS Code tip so everyone can see how cool you are 😎
 
 ## The Heisensofties: @ltalirz, @crazy4pi314, @guenp
+## HTRI: the version control tools are neat @noahbittner
```

Comment:
Can we move this to the end of the yearbook? Right now it's inserted in another entry! :)

Reply:
Done!